index.js

```diff
@@ -0,0 +1,39 @@
+function isLowerCase(password, index) {
+  return password.charCodeAt(index) >= 97 && password.charCodeAt(index) <= 122
+}
+
+function isUpperCase(password, index) {
+  return password.charCodeAt(index) >= 65 && password.charCodeAt(index) <= 90
+}
+
+function isANumber(password, index) {
+  return !isNaN(password[index])
+}
+
+function validatePassword(password) {
+  if (password.length < 8)
+    return false
+
+  let lower = 0
+  let Upper = 0
```

Comment:
Watch your casing, looks like JavaScript might be trying to treat that as a key word